templates/todo/projects/nodejs-mongo-swa-func/.repo/bicep/infra/main.bicep

```diff
@@ -144,6 +148,33 @@ module monitoring '../../../../../../common/infra/bicep/core/monitor/monitoring.
   }
 }
 
+// Creates Azure API Management (APIM) service to mediate the requests between the frontend and the backend API
+module apim '../../../../../../common/infra/bicep/core/gateway/apim.bicep' = if (useAPIM) {
+  name: 'apim-deployment'
+  scope: rg
+  params: {
+    name: !empty(apimServiceName) ? apimServiceName : '${abbrs.apiManagementService}${resourceToken}'
+    location: location
+    tags: tags
+    applicationInsightsName: monitoring.outputs.applicationInsightsName
+  }
+}
+
+// Configures the API in the Azure API Management (APIM) service
+module apimApi '../../../../../common/infra/bicep/app/apim-api.bicep' = if (useAPIM) {
+  name: 'apim-api-deployment'
+  scope: rg
+  params: {
+    name: useAPIM ? apim.outputs.apimServiceName : ''
```

Comment:
Do we need to check `useAPIM` again here?  (since we are using the `if (useAPIM)` above